db/table_cache.cc

```diff
@@ -404,7 +404,91 @@ Status TableCache::MultiGet(const ReadOptions& options,
   Status s;
   TableReader* t = fd.table_reader;
   Cache::Handle* handle = nullptr;
-  if (s.ok()) {
+  MultiGetRange table_range(*mget_range, mget_range->begin(), mget_range->end());
+#ifndef ROCKSDB_LITE
+  autovector<std::string, MultiGetContext::MAX_BATCH_SIZE> row_cache_entries;
+  IterKey row_cache_key;
+  size_t row_cache_key_prefix = 0;
+  KeyContext& first_key = *table_range.begin();
+  bool lookup_row_cache = ioptions_.row_cache &&
+          !first_key.get_context->NeedToReadSequence();
```

Comment:
Is NeedToReadSequence() guaranteed to return the same result for all GetContexts? Same question for has_callback() below.

Reply:
Yes. At present, getting internal sequence number is not supported in MultiGet(), so NeedToReadSequence() will always return false. We may have to revisit in future when we support ```MultiGetForUpdate()``` in TransactionDB. The has_callback() is also guaranteed to return the same value for all keys, since a single callback is passed to ```DBImpl::MultiGet()```.